### DISCUSSION TITLE
Drop (reference to) Python 2.7 support?

### DISCUSSION THREAD

**Discussion by mdxs** [ThomasGerstenberg/blatann#155] [General]
Probably the source code (and thus the library) already dropped support for Python 2.7 as the `setup.py` already specifies Python 3.7+ to be required.

However, in the README there is still some reference to Python 2.7 and an older version of Blatann being compatible in some combinations.

It probably makes sense to drop Python 2.7 support and thus the references in the README as well. Agree?

**Comment by ThomasGerstenberg** [comment]
Thanks for the suggestion. I removed reference to Python 2.7 in the readme, but I'll leave it in the docs in case there still are unfortunate souls stuck on 2.7
